libs/blocks/tabs/tabs.css

```diff
@@ -49,49 +64,68 @@
   padding: var(--spacing-xxl) 0;
 }
 
-.tabs.xxl-spacing .paddle {
+.tabs:not(.quiet).xxl-spacing .paddle {
   top: var(--spacing-xxl);
 }
 
+.tabs.quiet.xxl-spacing .paddle {
+  top: calc(var(--spacing-xxl) + 1px);
```

Comment:
Is there no way around the `+ 1px` repetition? Could we add a `1px` top margin to `.tabs.quiet`. Could we translate them by `1px`? I feel like we may be able to improve this a bit.